src/platform-includes/session-replay/setup/javascript.mdx

```diff
@@ -46,3 +46,72 @@ Sentry.init({
+  There is currently no PII scrubbing in canvas recordings!
+</Alert>
+
+<Alert level="">
+  Canvas recording is in beta and currently only available to{" "}
+  <a href="/product/accounts/early-adopter-features/">early adopters</a>
+</Alert>
+
+In order to record HTML canvas elements, you will need to add an additional integration in your Sentry configuration. The canvas integration is exported from the browser SDK, so no additional package is required. Canvas recording is opt-in and will be tree-shaken from your bundle if it is not used:
+
+```javascript
+// import Sentry from your framework SDK (e.g. @sentry/react) instead of @sentry/browser
+import * as Sentry from "@sentry/browser";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  replaysSessionSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+
+  integrations: [
+    // Keep the Replay integration as before
+    new Sentry.Replay(),
+
+    // The following is all you need to enable canvas recording with Replay
+    new Sentry.ReplayCanvas(),
+  ],
+});
+```
+
+```html {tabTitle: CDN}
+<!-- Existing script tag for bundle with replay, error, and performance monitoring -->
+<script
+  src="https://browser.sentry-cdn.com/{{@inject packages.version('sentry.javascript.browser') }}/bundle.tracing.replay.min.js"
+  integrity="sha384-{{@inject packages.checksum('sentry.javascript.browser', 'bundle.tracing.replay.min.js', 'sha384-base64') }}"
+  crossorigin="anonymous"
+></script>
+
+<!-- New script tag for the canvas replay integration -->
+<script
+  src="https://browser.sentry-cdn.com/{{@inject packages.version('sentry.javascript.browser') }}/replay-canvas.min.js"
+  integrity="sha384-{{@inject packages.checksum('sentry.javascript.browser', 'replay-canvas.min.js', 'sha384-base64') }}"
+  crossorigin="anonymous"
+></script>
+```
+
+#### Canvas Recording Performance
+
+The canvas recording integration works by exporting the canvas as an image (at a rate of 2 frames per second). However, in order to export images from 3D and WebGL canvases, the integration needs to enable `preserveDrawingBuffer` which can negatively affect canvas performance. If your canvas application is impacted by enabling `preserveDrawingBuffer`, then you will need to enable manual snapshotting and call a `snapshot()` method inside of your re-paint loop. There are two things you must do in order to use manual snapshotting: first you must enable it when initializing the `ReplayCanvas` integration:
+
+```javascript
+new Sentry.ReplayCanvas({
+  // Enabling the following will ensure your canvas elements are not forced
+  // into `preserveDrawingBuffer`.
+  enableManualSnapshot: true,
+});
+```
+
+Next, you need to call the the following `snapshot()` method inside of your application's paint loop. `snapshot()` needs to be called in the same execution loop as the canvas draw commands, otherwise you may be snapshotting empty canvas buffers. This is due to how WebGL works when `preserveDrawingBuffer` is `false`.
+
```

Comment:
```suggestion
**Step 2**
Call the the following `snapshot()` method inside your application's paint loop. `snapshot()` needs to be called in the same execution loop as the canvas draw commands, otherwise you may be snapshotting empty canvas buffers. This is due to how WebGL works when `preserveDrawingBuffer` is `false`.

```